tm2/pkg/telemetry/metrics/metrics.go

```diff
@@ -16,30 +16,30 @@ import (
 )
 
 const (
-	broadcastTxTimerKey = "broadcast_tx_hist"
```

Comment:
Please make sure to update the `misc/telemetry` Grafana dashboard values as well